bw_projects/filesystem.py

```diff
@@ -1,30 +1,48 @@
 import hashlib
+import re
 import os
+import unicodedata
 
+from typing import Union
 from pathlib import Path
 
 
-def create_dir(dirpath):
+re_slugify = re.compile(r"[^\w\s-]", re.UNICODE)
+SUBSTITUTION_RE = re.compile(r"[^\w\-\.]")
+MULTI_RE = re.compile(r"_{2,}")
+
+
+def safe_filename(
+    string: Union[str, bytes], add_hash: bool = True, full: bool = False
```

Comment:
Maybe this is covered elsewhere, but I would drop`full` and just make it always effectively `False`.

Reply:
Thank you for the suggestion, I will include this suggestion in [other PR](https://github.com/baali/bw_projects/pull/13) where I have introduced this function.

I rebased this PR on another open one and that's why the confusion.